### ISSUE TITLE
False positive legitimate kernel32.dll and pcasvc.dll

### ISSUE THREAD

**Issue by St-elin** [Neo23x0/signature-base#187]
Thx for a great tool. It's really handy for a rapid response. But there are too much noise from legitimate kernel32.dll and pcasvc.dll detection.

```
20220531T13:08:18Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-a..atibility-assistant_31bf3856ad364e35_6.1.7601.18526_none_35c509df816f8375\pcasvc.dll SCORE: 65 TYPE: UNKNOWN SIZE: 157184 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282EDC0> MD5: 3a55d53687f16d9ef5bf307bbfefcd9c SHA1: 0717d53b0574f2aaa51ab92e9d4b186645d64a24 SHA256: f1bb1b43442b151686500768c43a4d20caa47427e78386953a42ddb42d9ddf0c CREATED: Wed Oct 15 08:31:03 2014 MODIFIED: Mon Jul  7 04:40:18 2014 ACCESSED: Wed Oct 15 08:31:03 2014 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:08:18Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-a..atibility-assistant_31bf3856ad364e35_6.1.7601.18741_none_35aa6c61818436f2\pcasvc.dll SCORE: 65 TYPE: UNKNOWN SIZE: 157184 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282ED60> MD5: 52954be460ec6c54c0acb2b3b126ffc6 SHA1: ec0c63ee39e649b22cb568404eb07b771955f32b SHA256: 9f9878ec5abc74c5a8ee8e1d940f0934f081895b07d844f42f80a638fe713f7b CREATED: Wed Mar 11 08:17:44 2015 MODIFIED: Tue Feb  3 06:12:28 2015 ACCESSED: Wed Mar 11 08:17:44 2015 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:08:18Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-a..atibility-assistant_31bf3856ad364e35_6.1.7601.22736_none_3643d8a49a953c57\pcasvc.dll SCORE: 65 TYPE: UNKNOWN SIZE: 157184 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282EDC0> MD5: 14bafefa6706b520a17ef215cad3f9a3 SHA1: 181aacc1385605d53224fe20b0743310ef59229c SHA256: 6f6bbe095c91c0c2d0060258d5cbd600508f6647e8f5d09c1b0620dcbbf483e0 CREATED: Wed Oct 15 08:31:03 2014 MODIFIED: Mon Jul  7 04:40:54 2014 ACCESSED: Wed Oct 15 08:31:03 2014 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:08:18Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-a..atibility-assistant_31bf3856ad364e35_6.1.7601.22948_none_363b0d189a9b8544\pcasvc.dll SCORE: 65 TYPE: UNKNOWN SIZE: 157184 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282ED60> MD5: adedf4b70086bceaeebcf75578000d69 SHA1: 709026ea59c38ccc7b4aba352d6327e562312ad5 SHA256: 7d3e456e1af772d793b135a52c2ae4bfb998ae0d7b1e0d06bd500a0797b72b9a CREATED: Wed Mar 11 08:17:44 2015 MODIFIED: Tue Feb  3 06:32:02 2015 ACCESSED: Wed Mar 11 08:17:44 2015 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.17514_none_95c54f2cb48da1b9\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 857600 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AA30> MD5: 5553784d774ca845380650e010bbda2c SHA1: 9c1bcf751682ab4d36c8cd252a954828fb935f54 SHA256: ff4c354d08c8a6a6554de195c72e5c940be0def803ce0be8312161943a0f5821 CREATED: Sun Nov 21 00:29:19 2010 MODIFIED: Sun Nov 21 00:29:19 2010 ACCESSED: Sun Nov 21 05:41:11 2010 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.17651_none_95971084b4b0c29f\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AAF0> MD5: e570cbd732848438eac574eb3442a2a8 SHA1: b1df150923d905ce7803c3d43a6086b04d8b7dbf SHA256: b53eeffe97f96cde3d961fdad1a0ba4d5b25a273e282727292a7cb0d8f3ace9e CREATED: Tue Aug 14 10:14:39 2012 MODIFIED: Sat Jul 16 07:27:30 2011 ACCESSED: Tue Aug 14 10:14:39 2012 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.17932_none_95adb658b49f9b89\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282A958> MD5: 6f93a0f455963dc8a9a16bb682c8d589 SHA1: 1d5bcbcb1f00efdd60293c61364a6b0ffa362bc0 SHA256: 554d610b30ac06081b0e86c918460fd0ebf41a465eb6a12a04e119c004494ee1 CREATED: Thu Oct 11 07:08:02 2012 MODIFIED: Mon Aug 20 20:40:01 2012 ACCESSED: Thu Oct 11 07:08:02 2012 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.17965_none_95904772b4b53b61\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AA30> MD5: 3ed262888758e350c29e02207af9ac59 SHA1: 1cad5630968fcf4104ef46bb9798b647a50701df SHA256: d2ef50b6a2623148e7d67c96916cdcedb875c4f9fc8cb8617beee05e5110b893 CREATED: Wed Dec 12 07:19:22 2012 MODIFIED: Thu Oct  4 19:43:05 2012 ACCESSED: Wed Dec 12 07:19:22 2012 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.18015_none_95c62f30b48ce2ee\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AAF0> MD5: ae09b85158c66e2c154c5c9b3c0027b3 SHA1: ea174348498eaefeba3916a024e5cdd47c0ded68 SHA256: d6c98fdff67cd0353a0a93c8e96d92b69baeb54e19f9b16d359646826d18029c CREATED: Thu Jan 10 07:20:48 2013 MODIFIED: Fri Nov 30 07:47:44 2012 ACCESSED: Thu Jan 10 07:20:48 2013 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.18229_none_95bf6438b4915e89\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282A958> MD5: 6933e2aff444a7a95d5c67e98449163e SHA1: 70b8460dc96f914709b34ae76638261475e94005 SHA256: 4e745b89d319ff997f7dfd288e9d02143cef5474d2b8814803504a6570a146de CREATED: Thu Sep 12 07:23:32 2013 MODIFIED: Fri Aug  2 04:49:19 2013 ACCESSED: Thu Sep 12 07:23:32 2013 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.18409_none_95d507dcb48120f5\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AA30> MD5: f74ffa7654702f81884bdb41eb80dac2 SHA1: 83343001edfa431e3e441c012621f3e2bab3980c SHA256: 96901aa7a9e38c537bebbf3459eb129a654709ce21a143a080d50b021fea8626 CREATED: Wed Apr  9 07:06:07 2014 MODIFIED: Tue Mar  4 12:17:13 2014 ACCESSED: Wed Apr  9 07:06:07 2014 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:02Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.18847_none_95a7cf30b4a352a7\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AAF0> MD5: 957655757f43858692289b96f73716d8 SHA1: 7dd86605bf8ee08220b6e16ebdffcabaac1fb68b SHA256: 544da3d99d91ae823edd84a70c21048d879ce51af238ef61d37517857961b751 CREATED: Wed Jun 10 08:18:27 2015 MODIFIED: Sat May  9 06:13:42 2015 ACCESSED: Wed Jun 10 08:18:27 2015 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.19131_none_95ac794eb4a0cc3c\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 872448 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282A958> MD5: 7ea71253a1049c0d2c3f656fa0760d6e SHA1: edff8fb82a4648063e878046f0cf88dbd9875a9e SHA256: a37311bdc115136f74218edbb8fdebb55f4f9c5e803e4ba4129b382da723bf80 CREATED: Wed Feb 24 09:44:47 2016 MODIFIED: Sat Jan 16 21:35:36 2016 ACCESSED: Wed Feb 24 09:44:47 2016 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.19135_none_95b07a76b49d3198\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 872448 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AA30> MD5: 2362b7281a39807f1aa3550333a194bc SHA1: 3b7b2d77f4e5b10f61a33e8d0bd6cb3eb4c19203 SHA256: d80f773d2db0f1cbffb74b8571d084d9894bf57490f98a0019dc9447d0fcf637 CREATED: Wed Feb 24 09:44:42 2016 MODIFIED: Fri Jan 22 09:02:00 2016 ACCESSED: Wed Feb 24 09:44:42 2016 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.21772_none_960c0dc1cdddb3a2\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AAF0> MD5: 921f8b3ff01501c9934ccb3c270833d7 SHA1: 6abe61729146f88479516138056da50a875a6ab8 SHA256: 5ab1842b89b2820997e51b204ceed0c278bc6a901beefea94da6cea96790870e CREATED: Tue Aug 14 10:14:39 2012 MODIFIED: Sat Jul 16 07:54:28 2011 ACCESSED: Tue Aug 14 10:14:39 2012 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.22091_none_95f5498dcdeeffbd\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282A958> MD5: 9139b25aa9ca8749a11f2be863ef391b SHA1: 5dd7de01c847cb7b4383513b4fe2ab466e09c36d SHA256: 405400c932613010221fae127eaac518a37ef267a202952764c788bb7e3dab58 CREATED: Thu Oct 11 07:08:02 2012 MODIFIED: Mon Aug 20 20:34:45 2012 ACCESSED: Thu Oct 11 07:08:02 2012 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.22125_none_9644fc0fcdb29ea9\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AA30> MD5: 63350392c018d28c87e6fcb638dfcfe8 SHA1: d24179cbf279ba5dea528d2a040fa70a3f4b904c SHA256: 656b73cd5510b6d3033b40cfdcb7fdd2ab66ccec3fef92b9b050bab5ae5ccfb6 CREATED: Wed Dec 12 07:19:22 2012 MODIFIED: Thu Oct  4 19:32:16 2012 ACCESSED: Wed Dec 12 07:19:22 2012 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.22177_none_9610ed07cdd95d0c\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AAF0> MD5: 6d0d4b00c7cb4fa829f396a83b327894 SHA1: d22f7ef90bc413e4730c9f8a8f0d2348026663c0 SHA256: 2173b50ee2b931172c5069b74e40a73fe13aaf3b0dd4c72edb7567446848839b CREATED: Thu Jan 10 07:20:49 2013 MODIFIED: Fri Nov 30 08:01:46 2012 ACCESSED: Thu Jan 10 07:20:49 2013 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.22209_none_965e9ef5cd9ec94a\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282A958> MD5: f14125f0b2acb29963e896e3441dc30c SHA1: 14a8714f1a2e7408265edd48087cc0deed2f1029 SHA256: c193e493aca14a04de366cbb3f13e694504c64716ea9f6e31023bbe7759b8272 CREATED: Thu Feb 14 07:12:56 2013 MODIFIED: Fri Jan  4 07:46:46 2013 ACCESSED: Thu Feb 14 07:12:56 2013 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.22411_none_964bd085cdae14d1\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 868352 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AA30> MD5: 071350d18f2abc93496040f44d44f592 SHA1: 5fc33dba58c9b8e33ce2447a04dac6261b67a602 SHA256: 8dd3d9ec0ea67f26f9ba1dbcfe1775bf8e313bfaa917ae0c5f7af21977d788ab CREATED: Thu Sep 12 07:23:33 2013 MODIFIED: Fri Aug  2 08:54:18 2013 ACCESSED: Thu Sep 12 07:23:33 2013 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:03Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.22616_none_9650d5c3cda98dd2\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 872448 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AAF0> MD5: 8237bf64fdd5ff36985070b8ebef144d SHA1: a39897965b0dc34791e477a733c02366a7d9d2a0 SHA256: 8149d084504be6dbbe2760935976c1f1722ec9a29b521575cd01dd0cb7bfcb06 CREATED: Wed Apr  9 07:06:07 2014 MODIFIED: Tue Mar  4 13:39:13 2014 ACCESSED: Wed Apr  9 07:06:07 2014 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:04Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.22653_none_96229535cdccb191\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 872448 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282A958> MD5: 0acc3056081e646e242a8eab2348271a SHA1: 76e0d13679d846423337e607ff60404e29d7e452 SHA256: 1e677eba0064145ef5459e74e9549b36cca5b5bf6b739ebfb92a9686039d3497 CREATED: Wed May 14 07:13:19 2014 MODIFIED: Sat Apr 12 05:06:41 2014 ACCESSED: Wed May 14 07:13:19 2014 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:04Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.23049_none_963344b1cdbf5861\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 872448 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AA30> MD5: 8d5cc74bfa8f947cb283527806db7b1f SHA1: cd105b4741861cb592eeec795033a101aedccbb8 SHA256: 9a52185a4f532ab75d9ebeafacb94af94e2b4fa40620d59366b6ca885bd3e8b8 CREATED: Wed Jun 10 08:18:27 2015 MODIFIED: Sat May  9 08:41:31 2015 ACCESSED: Wed Jun 10 08:18:27 2015 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:04Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.23334_none_963918ddcdbbb532\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 872448 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282AAF0> MD5: 9b5e09d2ba0986cc915ead5633f0ca30 SHA1: 1f32cc2f5dbf36ee60a359de99555f9aceb8180d SHA256: 14ddac73a1e88c6e271725891bcdd8bf37bae84d5df6012b90760c3208c2413f CREATED: Wed Feb 24 09:44:47 2016 MODIFIED: Sun Jan 17 03:13:18 2016 ACCESSED: Wed Feb 24 09:44:47 2016 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
20220531T13:12:04Z ZAXAROVAI LOKI: Warning: MODULE: FileScan MESSAGE: FILE: C:\Windows\winsxs\x86_microsoft-windows-kernel32_31bf3856ad364e35_6.1.7601.23338_none_963d1a05cdb81a8e\kernel32.dll SCORE: 65 TYPE: UNKNOWN SIZE: 872448 FIRST_BYTES: 4d5a90000300000004000000ffff0000b8000000 / <filter object at 0x0282A958> MD5: 250afe0c57c1a0be64c81a7c53903b90 SHA1: 878fe620e78900424a36333ea58ad95d063ac6f0 SHA256: a8d3e7ef558812ce72ec7e231492e62c376b22cb20894735ccc2bad8ad9f12a2 CREATED: Wed Feb 24 09:44:42 2016 MODIFIED: Fri Jan 22 09:07:56 2016 ACCESSED: Wed Feb 24 09:44:42 2016 REASON_1: Yara Rule MATCH: SUSP_NullSoftInst_Combo_Oct20_1 SUBSCORE: 65 DESCRIPTION: Detects suspicious NullSoft Installer combination with common Copyright strings REF: https://twitter.com/malwrhunterteam/status/1313023627177193472 AUTHOR: Florian Roth MATCHES: Str1: NullsoftInst Str2: Microsoft Corporation
```

**Comment by pH-T** [comment]
hi @St-elin!
Im not able to replicate your finding:
the rule in question is https://github.com/Neo23x0/signature-base/blob/master/yara/gen_anomalies_keyword_combos.yar - whether I run this rule with yara or if I scan with thor-lite, it is not matching the mentioned files.
I looked into the history of this rule and I found this commit: https://github.com/Neo23x0/signature-base/commit/10d608c7386fda6eac662312efc84b884af70e59 - this commit seems to fix the mentioned FPs. If I remove the $fp1 string from the current rule it _does_  match. Is it possible that you are using a rather old ruleset?
